templates/default/slurm.conf.erb

```diff
@@ -69,10 +69,11 @@ SchedulerType=sched/backfill
 SelectType=select/cons_res
 SelectTypeParameters=CR_CPU
 
+# FastSchedule is deprecated in release 19.05.3-x
```

Comment:
let's just remove the part about FastSchedule and do that in a separate commit that contains this explanation